build/themes/default.html

```diff
@@ -1,1093 +1,390 @@
 <style>
-/* From https://gist.github.com/dashed/6714393 with Manubot modifications */
 
-/*! normalize.css v2.1.3 | MIT License | git.io/normalize */
+    /* import google fonts */
+    @import url("https://fonts.googleapis.com/css?family=Open+Sans:400,600,700");
+    @import url("https://fonts.googleapis.com/css?family=Source+Code+Pro");
```

Comment:
Do we have to worry about the dependency on Google? For example, doesn't china great :fire: :moyai: block google domains? Would it make sense to host these fonts on https://github.com/greenelab/manubot-resources/ (license permitting).

Reply:
Do they completely block all google.com? I wasn't aware of that. Will they somehow block the whole html document if they see one link to google there?

If they just block the font import, then it will fall back to helvetica, and if the user doesn't have that installed, it'll fall back to whatever sans-serif they have (usually arial).

https://github.com/greenelab/manubot-rootstock/blob/e4b9996a3f9c53207596b9198b1524c2542a6835/build/themes/default.html#L14

Reply:
So would it be possible to add the font files to `manubot-resources` and only use those if the googleapis import failed? It looks like they both have licenses that should allow this.

Reply:
Actually it looks like github doesn't support this:
https://stackoverflow.com/questions/39693837/how-do-i-use-github-to-host-a-webfont

=/

Reply:
I think a workaround would be to set GitHub pages to deploy the `master` branch of `manubot-resources`. One downside is that we wouldn't be able to provide versioning by commit hash then. Ideally, we would keep all links to repos versioned such that they are less likely to break.

Reply:
Or on second thought, we could make a `gh-pages` branch that stores versioned files and we commit to not removing them. We use the custom domain `resources.manubot.org` such that we could even switch web hosts.

Reply:
Would we be able to get the resource from `gh-pages` properly? Github can host fonts just fine, but when trying to link to the "raw" file, it doesn't work. According to that stackoverflow question, the reason is that gh raw files only support the text mime type.

Reply:
Is the font issue blocking this merge? I'm thinking `gh-pages` won't solve the issue if regular branches can't provide the right mime type. We can always add in our own hosting location of the fonts later. The only change that would be required would be adding an extra `@import url()` statement below the google ones.

Reply:
I think the GitHub Pages would properly serve the font files as it [seems to be](https://help.github.com/articles/mime-types-on-github-pages/) smart with mime types and is intended for hosting.

Not a blocker for merging, as we can always change this later. I think we potentially should, especially since some users do not like the omnipresence of google and would prefer not to use them if possible. In addition, to any potential issues with googleapis being blocked in china.